rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/rename/Modules.rsc

```diff
@@ -60,7 +126,7 @@ private bool isReachable(PathConfig toProject, PathConfig fromProject) =
 list[TextEdit] getChanges(loc f, PathConfig wsProject, rel[str oldName, str newName, PathConfig pcfg] qualifiedNameChanges) {
     list[TextEdit] changes = [];
 
-    start[Module] m = parseModuleWithSpacesCached(f);
+    start[Module] m = parseModuleWithSpaces(f);
```

Comment:
shouldn't this use the `Renamer` parser cache?

Reply:
This "module renaming" triggered by moving files is mostly separate and does not use the rename framework, so it also does not have access to the renamer cache.